templates/sushi_script.erb

```diff
@@ -11,15 +11,18 @@ if [[ -z "$vhost" ]]; then
 	exit 2
 fi
 
+# Remove non-alphanumeric characters
+VHOST_CLEAN=${vhost//[^a-zA-Z0-9]/}
```

Comment:
Need to include `-` and `_` characters

Reply:
These characters shouldn't be needed as they are stripped by dash - https://github.com/silverstripeltd/dash/blob/master/deploynaut/code/model/DNCodeGenerator.php#L18

Reply:
Apologies just realised this was for CCL which does support those characters